src/views/checkups/storage/details/CheckupsStorageDetailsPageSection.tsx

```diff
@@ -0,0 +1,216 @@
+  STORAGE_CHECKUPS_VM_VOLUME_CLONE,
+  STORAGE_CHECKUPS_WITH_CLAIM_PROPERTY_SETS,
+  STORAGE_CHECKUPS_WITH_NON_RBD_STORAGE_CLASS,
+} from '../utils/utils';
+
+type CheckupsStorageDetailsPageSectionProps = {
+  configMap: IoK8sApiCoreV1ConfigMap;
+  job: IoK8sApiBatchV1Job;
+};
+
+const CheckupsStorageDetailsPageSection: FC<CheckupsStorageDetailsPageSectionProps> = ({
+  configMap,
+  job,
+}) => {
+  const { t } = useKubevirtTranslation();
+
+  return (
+    <>
+      <Title className="co-section-heading" headingLevel="h2">
+        {t('Storage checkup details')}
+      </Title>
+      <Grid>
+        <GridItem span={6}>
+          <DescriptionList>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('Name')}</DescriptionListTerm>
+              <DescriptionListDescription>{configMap?.metadata?.name}</DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('Status')}</DescriptionListTerm>
+              <CheckupsNetworkStatusIcon configMap={configMap} job={job} />
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('Start time')}</DescriptionListTerm>
+              <DescriptionListDescription>
+                <Timestamp timestamp={configMap?.data?.[STATUS_START_TIME_STAMP] || NO_DATA_DASH} />
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('Default storage class')}</DescriptionListTerm>
+              <DescriptionListDescription>
+                {configMap?.data?.[STORAGE_CHECKUP_DEFAULT_STORAGE_CLASS] ? (
+                  <ResourceLink
+                    groupVersionKind={modelToGroupVersionKind(StorageClassModel)}
+                    name={configMap?.data?.[STORAGE_CHECKUP_DEFAULT_STORAGE_CLASS]}
+                  />
+                ) : (
+                  t('None')
+                )}
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>
+                {t('StorageProfiles with spec claimPropertySets')}
+              </DescriptionListTerm>
+              <DescriptionListDescription>
+                {configMap?.data?.[STORAGE_CHECKUP_LIVE_MIGRATION] || t('None')}
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('Storage missing VolumeSnapShootClass')}</DescriptionListTerm>
+              <DescriptionListDescription>
+                {configMap?.data?.[STORAGE_CHECKUPS_MISSING_VOLUME_SNAP_SHOT] || t('None')}
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>
+                {t('VirtualMachine with non virt RBD StorageClass')}
```

Comment:
Should this be 'non-virt RBD' or 'non-RBD' as it is in the property name?

Reply:
lets go with non-virt RBD , if needed will be changed according to doc guys